### DISCUSSION TITLE
Rating doesn't have Cancel Icon anymore

### DISCUSSION THREAD

**Discussion by RBedbur** [primefaces/community#1667] [PrimeVue] [answered]
HI,

in the rating component of Version 4, there is no Cancel Icon rendered anymore. The docu still mentions it in the template section, but it doesn't render.
I guess it's by design, as the rating is now removed, when an icon is clicked a second time?

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

Thanks for your attention. The demo will be fixed.
